socotratools/socotratools/client.py

```diff
@@ -284,3 +284,17 @@ def preview_endorsement_price(self, policy_locator, endorsement_name,
                 }
 
         return self.__post("/policy/" + policy_locator + "/previewEndorsementPrice", data)
+
+    def gen_report(self, report_name, start_timestamp, end_timestamp):
+        data = {'startTimestamp': start_timestamp,
+                'endTimestamp': end_timestamp
```

Comment:
This won't work for the unearned premium report

Reply:
Fixed.